### PR TITLE
reactjs proto that uses the javascript sdk

### PR DESCRIPTION
**Pull request by remicartierPTI**

### CODE REVIEW

File: examples/phase1-app/reactjs/.gitignore [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```
Comment (AlexBegin) [inline examples/phase1-app/reactjs/.gitignore:2]: I would probably move the content to its own new directory, the phase1-app is something that we did a long time ago, I think it should be like in "jssdkapp" or something like that cause the goal is to demonstrate that.

File: examples/phase1-app/reactjs/src/App.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  );
+}
+
+function SimpleDialog(props) {
+  const { open, callback, type } = props;
+  const [ sdkInit, setSdkInit ] = React.useState(false);
+  const ref = React.createRef(false);
+
+  React.useEffect(() => {
+        //setTimeout(() => callback(), 5000);
```
Comment (AlexBegin) [inline examples/phase1-app/reactjs/src/App.js:30]: Do we need that code?

File: examples/phase1-app/reactjs/public/index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="root"></div>
+    <script async defer src="https://sdk.apidev.pticlient.com/v1.0/index.js"></script>
+    <script>
+      window.ptiAsyncInit = function() {
+        PTI.init({
+          clientId: "clientId",
+          generateTokenPath: "/generateToken"
```
Comment (AlexBegin) [inline examples/phase1-app/reactjs/public/index.html:37]: Is the an endpoint for that, just a mock like?
What we could do that would be pretty cool is re-use the mvp demo app backend or something just to generate tokens, maybe we just pass the userid as a param... lets discuss later

**Review by AlexBegin — CHANGES_REQUESTED** [review]
I would probably move the content to its own new directory, the phase1-app is something that we did a long time ago, I think it should be like in "jssdkapp" or something like that cause the goal is to demonstrate that.

File: examples/phase1-app/reactjs/src/App.js (hunk 2 above)
Comment (remicartierPTI) [reply examples/phase1-app/reactjs/src/App.js:30]: Nope, can be removed

File: examples/phase1-app/reactjs/public/index.html (hunk 3 above)
Comment (remicartierPTI) [reply examples/phase1-app/reactjs/public/index.html:37]: the python mvp app has no endpoint to generateToken, the nearest I found was /token/{clientId} but it's a get

**Review by AlexBegin — APPROVED** [review]

**Pull request merged**